demo/index.html

```diff
@@ -28,9 +29,14 @@
         });
     </script>
     <script type="module" src="../src/auro-flight.js"></script>
-    <script src="https://unpkg.com/@alaskaairux/auro-accordion@latest/dist/auro-accordion__bundled.js" type="module"></script>
-    <script src="https://unpkg.com/@alaskaairux/auro-icon@latest/dist/auro-icon__bundled.js" type="module"></script>
-    <script src="https://unpkg.com/@alaskaairux/auro-alerts@latest/dist/auro-alerts__bundled.js" type="module"></script>
-    <script src="https://unpkg.com/@alaskaairux/auro-datetime@latest/dist/auro-datetime__bundled.js" type="module"></script>
+    <script type="module">
+      import { registerComponent } from "../index.js"
+      registerComponent('custom-flight');
+    </script>
```

Comment:
This is not needed?

Reply:
right, it is already imported in each of the partial demo files.